### ISSUE TITLE
Cannot delete default admin user

### ISSUE THREAD

**Issue by Tucker2015** [LibreTime/libretime#205]
Trying to remove default admin user after adding new Admin account and logging in but its not removing the admin account.  Comes up asking to remove user but still the user is present in the list.

Ubuntu 14.04
LibreTime 3.0.0-alpha.1

**Comment by hairmare** [comment]
It looks like re-assigning the users existing files to a new user is broken since it tries to [assign files to the first admin user](https://github.com/LibreTime/libretime/blob/2b30a14744f857657f2e3b7e9c96713929343dd2/airtime_mvc/application/controllers/UserController.php#L207). I'm preparing a fix that make LibreTime ignore that first user during the delete so the files get assigned to the next existing admin user.

**Comment by hairmare** [comment]
See #206 for the fix for this issue. As a workaround you can change your new user to be a superadmin allowing the initial non superadmin to be deleted.

```
# select all users to figure out what id your new user has
sudo -u postgres psql airtime -c "select id,login,type from cc_subjs;"

# set user to superadmin (replace the 2 at the end of this with your actual user id)
sudo -u postgres psql airtime -c "update cc_subjs set type = 'S' WHERE id = 2;"
```

Deleting the user should now work since superadmins have precendence when it comes to reassigning files. 

You may want to reset the superadmin back to plain admin after deleting the user (I think the main difference between admins and superadmins is that the latter can't reset their password or edit their details).

```
# replace the 2 at the end of this with your actual user id
sudo -u postgres psql airtime -c "update cc_subjs set type = 'A' WHERE id = 2;"
```

The patch to properly fix this should make it into `3.0.0-alpha.2`. Please let us know if you need any further assistance. Thanks for the bug report!

**Comment by hairmare** [comment]
The fix for this was merged to master in #206. Can you check with the latest master to confirm that the issue has been fixed?

**Comment by hairmare** [comment]
The fix for this has now been released in 3.0.0-alpha.2.
